js/dialog.js

```diff
@@ -28,9 +29,29 @@ export function updateDetailsDialog(data, cardUrl) {
     <p>Artist: ${data["Artist Credit"]} | Writer: ${data["Writer Credit"]}</p>
   `
   );
-  // Card
-  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].src =
-    cardUrl;
+  // Card art is applied
+  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].style.backgroundImage = 'url("' +
+    cardUrl + '")';
+
+  // Add animation start
+  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].classList.add(cardRarity(data["Rarity Folder"]), "animated");
+  
+  // Animation reset
+  DETAILS_DIALOG_EL.getElementsByClassName("card-details-dialog-close")[0].addEventListener("click", cardReset);
+  document.getElementsByClassName("dialog-overlay")[0].addEventListener("click", cardReset);
```

Comment:
I will need to revisit this later, I just had a go at swapping this out and it no longer works as expected. Yes, the animation relies on the appropriate classes being added to a the particular div that holds the card image; and this will hold on to the previous styles when the dialog is closed if not reset (feedback from Bing previously).

Reply:
Added task to Notion for followup later - no changes needed here for this PR

Reply:
I know you said to follow up later, but I have figured this out and updated. :)